packages/terra-props-table/src/terra-dev-site/test/props-table-test/MockPrivatePropsComponent.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// These props are used to test react-doc-gen, but they only need to be mocked
+/* eslint-disable react/no-unused-prop-types */
+const propTypes = {
+  /**
+   * An example of type string markup.
+   * `This component contains a very long section of code in it's description column. This component contains a very long section of code in it's description column.`
+   */
+  string: PropTypes.string.isRequired,
+  /**
+   * [Private] A private prop that should not show up in the rendered markup.
```

Comment:
This is what a private prop would look like in our other components.

Reply:
This looks clean. It's a little uglier but JSDoc has a tag for private variables. 

http://usejsdoc.org/tags-private.html

```
/**
 * @private
 */
privateProp: PropTypes.string,
```

Reply:
+1 for `@private`